### PR TITLE
move hardcoded pinecone variables to credentials

### PR DESCRIPTION
**Pull request by Nailan**
See issue #19

### CODE REVIEW

File: pages/credentials.tsx [hunk 1]
```diff
@@ -103,6 +116,30 @@ export default function CredentailsPage() {
...
                 </div>
 
+                <div className="grid grid-cols-4 items-center gap-4">
+                  <Label htmlFor="pinecone-environment" className="text-right">
+                    Pinecone Environment
+                  </Label>
+                  <Input
+                    id="pinecone-environment"
+                    value={pineconeEnvironment}
+                    className="col-span-3"
```
Comment (fraserxu) [inline pages/credentials.tsx:126]: When I first started to use pinecone, I didn't quite get what `environment` is. Would be good to put a `placeholder="us-west1-gcp"` here

File: pages/credentials.tsx [hunk 2]
```diff
@@ -103,6 +116,30 @@ export default function CredentailsPage() {
...
+                </div>
+
+                <div className="grid grid-cols-4 items-center gap-4">
+                  <Label htmlFor="pinecone-index" className="text-right">
+                    Pinecone Index Name
+                  </Label>
+                  <Input
+                    id="pinecone-index"
+                    value={pineconeIndex}
+                    className="col-span-3"
```
Comment (fraserxu) [inline pages/credentials.tsx:138]: same here be good to have a placeholder here

**Comment by fraserxu** [comment]
Thanks @Nailan , I left two small non-blocking suggestions.

**Review by fraserxu — APPROVED** [review]

File: pages/credentials.tsx (hunk 1 above)
Comment (Nailan) [reply pages/credentials.tsx:127]: makes sense, done

**Review by fraserxu — APPROVED** [review]

**Pull request merged**